tests/unit/challenges/test_views.py

```
@@ -4722,6 +4747,34 @@ def test_particular_challenge_phase_update(self):
         response = self.client.put(self.url, self.data, format="multipart")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
+    @override_settings(MEDIA_ROOT="/tmp/evalai")
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/evalai"

(S108)